frontend/components/panels/GrossProfitCheckPanel.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name" :disabled="!isExplanationsShown">
```

Comment:
I though the proposal from lukas was to still show text even in expert mode? I also find it sensible, so let's get rid of the `disabled` in those instances